### ISSUE TITLE
About the installation.

### ISSUE THREAD

**Issue by jeffkimkorea-git** [RozDavid/LOL#15]
I am trying to use the 'LOL' as well. 

But I could not even suceed to install it all. 

I would like to ask about the version that you have used.

Currently I am trying with the cuda 10.0 and cudnn 7.4.2. 

Installed tensorflow-gpu 1.14.0 and I could not finish the build. 

I didn't not used the virtual environment.  

Could you give any kind of help please? 

Thanks for your work.

**Comment by RozDavid** [comment]
Hey @Jeff-kookmin, 

The original code was built with Cuda 9.0, ROS Kinetic and Ubuntu 16.04. 
If you rather not change anything I would suggest you to roll back to the specified Cuda version that should certainly work out of the box.

For different cuda versions the challenging part is the the build of _tensorflow_ros_cpp_, for this I would suggest you to check out [this](http://wiki.ros.org/tensorflow_ros_cpp) page for guidance. 

Also, for more specific help please share your error log. 

Cheers, 
David

**Comment by jeffkimkorea-git** [comment]
Thanks for your kindly reply @RozDavid very much. 
I first tried to build the segmap as its core algorithm of LOL. I finally succeed to build it and run it. 
I could learn a lot more about the setting the version things and I am now 'finally' trying to build 'LOL' which is yours.
I thought it would be easier to build this since I already have suceed to build segmap. But it is not that easy as much as I have thought. :)

Thanks for letting me know what the challenging part is and let me continue to try building this LOL finally. Thanks a lot for the work again and let me comment for the extra errors from the installation. 

Sincerely,
Jeff
